edgelet/iotedge/src/check/checks/parent_hostname.rs

```diff
@@ -29,14 +27,8 @@ impl Checker for ParentHostname {
 
 impl ParentHostname {
     fn inner_execute(&mut self, check: &mut Check) -> Result<CheckResult, failure::Error> {
-        let settings = if let Some(settings) = &check.settings {
-            settings
-        } else {
-            return Ok(CheckResult::Skipped);
-        };
-
         let config_parent_hostname =
-            if let Some(config_parent_hostname) = settings.parent_hostname() {
+            if let Some(config_parent_hostname) = check.parent_hostname.as_ref() {
```

Comment:
This whole check needs to be moved into `aziotctl check`, right?

Reply:
Yes, it should. It's not pressing, but we should pop open a PBI to track this work. The bulk of the effort will just be coordinating the two PRs